readme.md

```diff
@@ -230,6 +242,6 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+find.Andy
+find.Resin_Laying_Deer_Figurine,Gold
 ```
```

Comment:
You should take a look at the Mongoose lesson plans and see what sort of syntax is used there. I have found it helpful to match Mongoose commands with their equivalents in Active Record.

Here is one possible answer for this question...

```js
Instructor.findOne({name: "Andy"), function(err, docs){
  docs.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold");
  docs.save();
});
```